examples/gnrc_border_router/Makefile

```diff
@@ -52,7 +52,7 @@ QUIET ?= 1
 
 # Ethos/native TAP interface and UHCP prefix can be configured from make command
 TAP ?= tap0
-IPV6_PREFIX ?= 2001:db8::/64
+IPV6_PREFIX ?= 2001:db8::/32
```

Comment:
This change has as a consequence GLOBAL addresses remaining TNT when using ethos:

```
2020-04-07 14:22:48,180 #           inet6 addr: fe80::3c8b:2187:3d8b:2186  scope: link  VAL
2020-04-07 14:22:48,187 #           inet6 addr: 2001:db8::3c8b:2187:3d8b:2186  scope: global  TNT[3]
```

Reply:
On which node? The BR itself?

Reply:
Ah damn, UHCP is advertising that prefix as is, not appending more bits like most DHCPv6 servers do :-/

Reply:
Seem like its an issue with uhcp:

```
uhcp: push from :57902 prefix=2001:db8:3500:0:7520:301f::/32
```

Reply:
> Ah damn, UHCP is advertising that prefix as is, not appending more bits like most DHCPv6 servers do :-/

I guess the proper fix would still bee addressing this in uhcp as well right?

Reply:
Be my guest :-)

Reply:
> Be my guest :-)

I'll but it on the todo-list :)